content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -47,8 +47,8 @@ Verify that zero padding is enabled on the PowerFlex storage pools that will be
 ### Install PowerFlex Storage Data Client
 
 The CSI Driver for PowerFlex requires you to have installed the PowerFlex Storage Data Client (SDC) on all Kubernetes nodes which run the node portion of the CSI driver. 
-SDC could be installed automatically by CSI driver install on Kubernetes nodes with OS platform which support automatic SDC deployment; for Red Hat CoreOS (RHCOS), RHEL 7.9 and RHEL 8.x. On Kubernetes nodes with OS version not supported by automatic install, you must perform the Manual SDC Deployment steps [below](#manual-sdc-deployment).
-Refer to https://hub.docker.com/r/dellemc/sdc for supported OS versions.
+SDC could be installed automatically by CSI driver install on Kubernetes nodes with OS platform which support automatic SDC deployment; for Red Hat CoreOS (RHCOS), RHEL 7.9, RHEL 8.4, RHEL 8.6. On Kubernetes nodes with OS version not supported by automatic install, you must perform the Manual SDC Deployment steps [below](#manual-sdc-deployment).
+Use this sdc image https://hub.docker.com/layers/dellemc/sdc/3.6.0.6/images/sha256-dcef857bc10039a9274b3ca042d0e7f50eb184166628a0ec27f11b885d88fe30?context=explore for Automated sdc deployment.
```

Comment:
I would not include a link to specific SDC version in docs, we will need to repeatedly update it each release

Reply:
done